index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
+filter, Kalman filter etc) is platform specific and can happen in software or hardware, ie. on a
+sensor hub.
 
 A [=latest reading=] per [=sensor=] of orientation type includes one [=map/entries=]
 whose [=map/key=] is "quaternion" and whose [=map/value=] contains a four element [=list=].
 The elements of the [=list=] are equal to components of a unit quaternion representing orientation
-of the device in a three dimensional Cartesian coordinate system (x, y, z), where x-axis points east,
-y-axis points north and z-axis points up and is perpendicular to the plane made up of x and y axis.
+of the device in a three dimensional Cartesian coordinate system (x, y, z), where the x-axis points
+east, the y-axis points north and the z-axis points upwards and is perpendicular to the plane made 
+up of x and y axis. The values are no necessarily normalized.
```

Comment:
yes, the vector has a length of one... but that definition probably differs a bit for quaternions as only the 3 latter parts represents the vector the the first part the scalar. I will dig up a definition somewhere